modules/110-istio/templates/kiali/rbac-for-us.yaml

```diff
@@ -95,10 +95,6 @@ rules:
   - tokenreviews
   verbs:
   - create
-- apiGroups: ["apps"]
```

Comment:
If you remove only the RBAC for Trickster, Kiali won't work with the aggregator proxy. You need to update the RBAC settings for Kiali to work with the aggregator proxy.